### DISCUSSION TITLE
Would anyone be interested in a paid version of this app with support?

### DISCUSSION THREAD

**Discussion by robvanderleek** [repository-settings/app#996] [Ideas]
This App works great and covers the major use-cases, yet feature requests for more niche use-cases seem to stack up.

Would anyone be interested in a paid version (small monthly subscription through Marketplace), that would allow for dedicated support and work on roadmap items?

**Comment by travi** [comment]
could you provide more context around your suggestion? are you suggesting that the maintainers of the project pursue a paid option that could provide income back to the project? from what you have posted, it sounds more like you are suggesting an option that is separate from the project, so it is not clear how it would be beneficial for the project. 

we, as the maintainers, have considered various options for funding/sponsorship/etc, but have not yet pursued such options because it is more complicated than income directly translating into more dedicated support time. before we would move forward with such options, we need to consider how it impacts the sustainability of the project.

**Comment by robvanderleek** [comment]
Sorry, I should have made it more clear that I was talking about a separate, paid, version of this app. A fork of the project that offers features that are requested by a small group of users (and therefore will never end up in this codebase), and offers dedicated support (for example predictable availability or different deployment models).

I'm curious if there are business users willing to pay a monthly fee for that.
